packages/frontend/src/app/modules/flow-builder/store/builder/flow-item-details/flow-items-details.effects.ts

```diff
@@ -52,7 +52,7 @@ export class FlowItemsDetailsEffects {
 						return new FlowItemDetails(
 							TriggerType.PIECE,
 							c.displayName,
-							`Trigger this flow following a specific event on ${c.displayName}`,
+							``,
```

Comment:
Please bring back the description, the UI will not look good and feel like something is missing.

Reply:
discusesed offline